### DISCUSSION TITLE
Is test-utils 2 supports Vue3 Script Setup State driven dynamic css?

### DISCUSSION THREAD

**Discussion by dergunovs** [vuejs/test-utils#1285] [Q&A] [answered]
https://v3.vuejs.org/api/sfc-style.html#state-driven-dynamic-css - when i try to test components, where this feature is present, even coverage do not collecting.

Failed to collect coverage
ERROR: Error parsing JavaScript expression: Bad character escape sequence.

Example in docs throws an error too.

**Comment by cexbrayat** [comment, marked as answer]
Hi @dergunovs 

We need additional information to help you. Can you provide a minimal reproduction in a Github repo? That way, we'll see the exact error you're having and which version of what library you are using.

Ideally, make a repo with a simple component and a failing test. You can use create-vue or Vue CLI to generate a project easily.

**Reply by dergunovs** [reply]
It was a very strange bug with directory naming in win 11. Just renamed the folder and everything became ok.
